README.md

```diff
@@ -2,30 +2,163 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Add fields (instance variables) to a class
+- Declare a variable with a class data type
+- Create an object in Java using the `new` operator
+- Assign a variable as an object reference
+- Use IntelliJ's Java Visualizer to view object state
+- Assign an instance variable an initial value at variable declaration
+- Use IntelliJ to auto-generate a `toString()` method for printing object state
+- Use dot notation to access and update object state
 
-## Introduction
+## Instructions
 
-A 1-2 sentence summary of what will be covered.
+The starter project contains a class named `Book` with a `main` method that contains comments outline the lab tasks.
 
-## Instructions
+You will use IntelliJ's debugger and the Java Visualizer to view object state.
+Recall that you can stop the debugger session at any point by pressing the red square icon:
+
+![stop debugger](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-oop-fundamentals/stop_debugger.png)
+
+You can also press the green arrow icon to continue execution to the next breakpoint
+or end of program:
+
+![continue execution](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-oop-fundamentals/resume_program.png)
+
+## Task #1 - Add fields and create 3 `Book` objects
+
+1. Edit the `Book` class to add 5 fields named `title`, `isbn`, `edition`, `pages`, and `language`.
+   The title, isbn, and language are strings.  The edition and pages are integers.
+2. Edit the `main()` method to create 3 instances of the `Book` class.  Declare local variables named
+   `book1`, `book2`, and `book3` and assign each variable to reference a new `Book` instances.
```

Comment:
"... and assign each variable to reference a new `Book` instances."

I believe "instances" should be singular: "instance".

Reply:
fixed